examples/feature_selection/plot_digits.py

```diff
@@ -0,0 +1,51 @@
+"""
+==========================================
+Feature selection for digit classification
+==========================================
+
+This examples show how different pixels of digits are ranked by recursive
+feature elimination and feature selection based on mutual information.
+
+.. note::
+
+    See also 
+    :ref:`example_feature_selection_plot_rfe_with_cross_validation.py`,
+    :ref:`example_feature_selection_plot_mutual_info.py`
+"""
+print(__doc__)
+
+import numpy as np
+from sklearn.svm import SVC
+from sklearn.datasets import load_digits
+from sklearn.feature_selection import RFE, MutualInfoSelector
+import matplotlib.pyplot as plt
+
+# Load the digits dataset
+digits = load_digits()
+X = digits.images.reshape((len(digits.images), -1))
+y = digits.target
+
+# Create the RFE object and rank each pixel
+svc = SVC(kernel="linear", C=1)
+rfe = RFE(estimator=svc, n_features_to_select=1, step=1)
+rfe.fit(X, y)
+ranking_rfe = rfe.ranking_.reshape(digits.images[0].shape)
+
+mrmr = MutualInfoSelector(n_features_to_select=X.shape[1],
+                          discrete_features=True, discrete_target=True)
+mrmr.fit(X, y)
+ranking_mrmr = mrmr.ranking_.reshape(digits.images[0].shape)
```

Comment:
Minor: This should be `mrmr.ranking_ + 1` by the current code. (But it will not matter much while displaying the plots)